rust/datafusion/tests/dataframe.rs

```diff
@@ -71,8 +71,7 @@ async fn join() -> Result<()> {
 
     let a = df1.join(df2, JoinType::Inner, &["a"], &["a"])?;
 
-    let batches = a.collect().await?;
-    assert_eq!(batches.len(), 1);
+    let _ = a.collect().await?;
```

Comment:
maybe we could change the test to assert on the total number of rows in the result set rather than the number of batches?

Reply:
Agreed 👍

Reply:
Done